src/ducks/transfers/TransferPage.jsx

```diff
@@ -303,7 +333,10 @@ class TransferPage extends React.Component {
       slide: 0,
       senderAccount: null,
       senderAccounts: [], // Possible sender accounts for chosen person
-      amount: ''
+      amount: '',
+      transferSent: false,
+      sendingTransfer: false,
+      transferError: null
```

Comment:
Nit: I think using a string and a single property would be better, so impossible states are impossible ©

Reply:
💯

Reply:
Adding a comment and solving this in a later PR.